xain_fl/coordinator/metrics_store.py

```diff
@@ -96,8 +134,42 @@ def write_metrics(self, metrics_as_json: str):
             validate(instance=metrics, schema=self.schema)
             self.influx_client.write_points(metrics)
         except Exception as err:  # pylint: disable=broad-except
```

Comment:
I guess we can't be more specific about expected exceptions from Influx here?

Reply:
Sadly the InfluxDB client documentation doesn't give much information about which exceptions can occur and I don't what that the coordinator crashes because of that.